gamestonk_terminal/portfolio_optimization/README.md

```diff
@@ -1,169 +1,192 @@
+* -v : If provided, this represents an actual allocation amount for the portfolio.  Defaults to 1, which just returns the weights.
+* --pie : Display a pie chart for weights
+
+![marketCap](https://user-images.githubusercontent.com/25267873/114958294-980a7700-9e5a-11eb-9db4-d06e6a1e8981.png)
+
+
+## Mean Variance Optimization <a name="Mean_Variance_Optimization"></a>
+
+These approaches are based off of the efficient frontier approach, which is meant to solve the following optimization problem.
+
+Minimize: <img src="https://latex.codecogs.com/svg.image?w^TS&space;w" title="w^TS w" />
+
+Subject to: <img src="https://latex.codecogs.com/svg.image?w^TR&space;>&space;R^*" title="w^TR > R^*" />, and <img src="https://latex.codecogs.com/svg.image?w_1&plus;w_2&plus;...w_n&space;=&space;1" title="w_1+w_2+...w_n = 1" />
+
+* Where S is the covariance matrix between stocks and R is the expected returns.  
+* The condition that all weights add up to 1 just implies that you want to have a net long portfolio (with no margin).  
+* A long-short portfolio can have negative weights and usually wants to have everything add up to 0 for a market-neutral strategy.
+
+Currently, we do not allow for changing risk models or adding constraints.  
+If there is something specific, please submit a feature request, or if you can write it, feel free to add a PR!
+All of these commands use [PyPortFolioOpt](#https://pyportfolioopt.readthedocs.io/en/latest/index.html) package.
+
+
+### maxsharpe <a name="maxhsarpe"></a>
+
 ````
-eff_risk [-p PERIOD] [-r --risk RISK_LEVEL] [-v --value VALUE] [--pie]
+usage: max_sharpe [-p {1d,5d,1mo,3mo,6mo,1y,2y,5y,10y,ytd,max}] [-v VALUE] [--pie] [-r RISK_FREE_RATE]
 ````
-* -p/--period Amount of time to retrieve data from yfinance. Options are: 1d,5d,1mo,3mo,6mo,1y,2y,5y,10y,ytd,max and it defaults to 3mo.
-* -r/--risk Risk tolerance.  Default is 0.1 (10%)
-* -v/--value If provided, this represents an actual allocation amount for the portfolio.  Defaults to 1, which just returns the weights.
-* --pie Flag that displays a pie chart of the allocations.
-
-### eff_ret
-This portfolio minimizes the risk at a given return level
-The usage is:
+
+Maximise the Sharpe Ratio. The result is also referred to as the tangency portfolio, as it is the portfolio for which the capital market line is tangent to the efficient frontier. This is a convex optimization problem after making a certain variable substitution. See Cornuejols and Tutuncu (2006) <http://web.math.ku.dk/~rolf/CT_FinOpt.pdf> for more. The sharpe ratio is defined as (Mean Returns - Risk Free Rate)/(Standard Deviation of Returns).
```

Comment:
Seems a little overkill?  I would drop the complex optimization sentence

Reply:
I just copied the one that the pyportfopt had for this funcionality. But I can shorten it